app/styles/main.scss

```diff
@@ -8,3 +8,28 @@
 .float-right {
   float: right;
 }
+
```

Comment:
I did not want to add this styling and am not very happy about it but it looks right and is the lesser of two evils.

If you look at how these tables are constructed, class's are slung in which makes it very difficult to create consistent spacing, reseting the marign/padding like this is a very easy way to do it right.